samTemplates/amazon-connect-interactive-messages-example/events/1_1_2_3-2020-10-31-T1800.json

```diff
@@ -0,0 +1,109 @@
+{
+    "requestAttributes": {
+        "x-amz-lex:accept-content-types": "PlainText"
+    },
+    "sessionId": "5d11c7ca-6eb1-48d1-b5f2-58510e9ae6c3",
+    "inputTranscript": "2020-10-31T18:00+00:00",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that your JSON object is invalid. Some possible reasons for this could be: use of single quotes, trailing commas, use of mathematical expressions etc. Make sure that you correctly format the JSON object to avoid production issues later on.



Similar issue at line numbers 24, 26, 75, and 77.